encrypt-cluster-comms/README.md

```diff
@@ -12,11 +86,42 @@ $ oc create \
   -f https://raw.githubusercontent.com/jboss-container-images/jboss-datagrid-7-openshift-image/1.1/templates/datagrid72-https.json
 ```
 
-```bash
-$ oc login -u developer
+## Creating a Project and Importing Secrets
+Create a new project, configure roles, and then import the HTTPS and JGroups keystores as OpenShift secrets.
 
-$ oc create \
-   -f https://raw.githubusercontent.com/jboss-openshift/application-templates/master/secrets/datagrid-app-secret.json
+1. Log in as the developer user and create a new project.
+  ```bash
+  $ oc login -u developer
 
-$ oc process openshift//datagrid72-https | oc create -f -
+  $ oc new-project rhdg-https
+  ```
+
+2. Add the view role to the default service account to allow access to resources in your project, which is required to manage the cluster.
```

Comment:
Is this really needed?

Reply:
honestly I'm not 100% on the step to add the view role. I thought it might be required for clustering based on other docs that I've seen. maybe we should remove it?

Reply:
It's normally required when you have apps that are going to cluster themselves. For server it should not be needed... Give it a go and see what happens.